mr/src/test/java/org/apache/iceberg/mr/hive/serde/objectinspector/TestIcebergObjectInspector.java

```diff
@@ -99,14 +98,9 @@ public void testIcebergObjectInspector() {
     assertThat(dateField.getFieldID()).isEqualTo(3);
     assertThat(dateField.getFieldName()).isEqualTo("date_field");
     assertThat(dateField.getFieldComment()).isEqualTo("date comment");
-    if (HiveVersion.min(HiveVersion.HIVE_3)) {
-      assertThat(dateField.getFieldObjectInspector().getClass().getName())
-          .isEqualTo(
-              "org.apache.iceberg.mr.hive.serde.objectinspector.IcebergDateObjectInspectorHive3");
-    } else {
-      assertThat(dateField.getFieldObjectInspector().getClass().getName())
-          .isEqualTo("org.apache.iceberg.mr.hive.serde.objectinspector.IcebergDateObjectInspector");
-    }
+    assertThat(dateField.getFieldObjectInspector().getClass().getName())
+        .isEqualTo(
+            "org.apache.iceberg.mr.hive.serde.objectinspector.IcebergDateObjectInspectorHive");
```

Comment:
seems that this is the wrong class here (and further below)?

Reply:
also why doesn't the unit test fail if the wrong class is being used here?

Reply:
I removed iceberg-mr from tests from removing the hive2 tests. Now it's added in hive3 tests while there are new test errors I need to fix. Thanks for catching this.